### PR TITLE
Use a set of test components in addon stories

### PR DESCRIPTION
**Pull request by tmeasday**
Telescoping on https://github.com/storybookjs/storybook/pull/18824

Issue: https://linear.app/chromaui/issue/SB-570/copy-in-the-generic-stories-based-on-chosen-addons-make-them-work-in

## What I did

- Use `globals.Components` to drive generic stories
- Add `globals.Components` via symlinked (JS-only) `template/components` directory inside each renderer.

image

image

## How to test

```
yarn example --framework react --force-delete
```

### CODE REVIEW

File: scripts/example.ts [hunk 1]
```diff
@@ -255,6 +261,12 @@ async function main() {
     const rendererStoriesDir = path.resolve(codeDir, `./renderers/${renderer}/src/stories`);
     await copyStories(rendererStoriesDir, path.join(storiesDir, renderer), { isTS });
 
+    // TODO: why can I not just use `@storybook/react/test-components`? @ndelangen?
```
Comment (tmeasday) [inline scripts/example.ts:264]: @ndelangen any idea why this wouldn't work? It seems like (according to `package.json:exports`) it points to the same place.

File: code/addons/actions/template/stories/basics.stories.tsx [hunk 2]
```diff
@@ -1,17 +1,10 @@
...
-// TODO -- this needs to be a generic component
-const Button: FC<{ onClick: () => void }> = ({ onClick, children }) => (
-  <button type="button" onClick={onClick}>
-    {children}
-  </button>
-);
-
 export default {
-  component: Button,
+  component: 'Button',
```
Comment (shilman) [inline code/addons/actions/template/stories/basics.stories.tsx:4]: @tmeasday is this a typo? shouldn't it be `global.Components.Button`?

Comment (tmeasday) [reply code/addons/actions/template/stories/basics.stories.tsx:4]: Must be some kind of merging/updating snafu. I thought I checked it was working before I pushed but I guess not. It definitely said that at one point ;)

**Comment by tmeasday** [comment]
Self merging @shilman

**Pull request merged**